__tests__/app.test.js

```diff
@@ -89,4 +89,48 @@ describe("/api/articles/:article_id", () => {
             expect(response.body.msg).toBe("Bad request")
         })
     })
+})
+describe("GET /api/articles/:article_id/comments", () => {
+    test("Status 200 : responds with an array of comments with the given article_id, sorted by date", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({ body: { comments } }) => {
+            expect(comments).toHaveLength(11);
+            comments.forEach(comment=>{
+                expect(comment).toHaveProperty("comment_id")
+                expect(comment).toHaveProperty("votes")
+                expect(comment).toHaveProperty("created_at")
+                expect(comment).toHaveProperty("author")
+                expect(comment).toHaveProperty("body")
+                expect(comment).toHaveProperty("article_id")
```

Comment:
We know the article id as you're looking for 1. We should assert that the id is correct as well.